language_detector/main.py

```diff
@@ -1,8 +1,14 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
+from .languages import LANGUAGES
```

Comment:
What is the use of this? Aren't the languages imported in the test file, and then passed through to our function anyway?